File: MenuDotNetMS/MenuDotNetMS/Models/fournisseurModel.cs
namespace MenuDotNetMS.Models
{
    public class fournisseurModel
    {
        public string id { get; set; } = string.Empty;
        public string? raison_social { get; set; }
        public string telephone { get; set; } = string.Empty;
        public string ice { get; set; } = string.Empty;
        public string? adresse { get; set; }
        public DateTime date_creation { get; set; }
    }
}

File: InterfaceAngular/src/app/menu/models/fournisseur.model.ts
export interface Fournisseur {
  id: string;
  ice: string;
  raisonSocial: string;  // Note: PascalCase comme la réponse
  telephone: string;
  adresse: string;
  dateCreation: Date;
}

export interface FournissueurAddDtoRequest {
  ice: string;
  raisonSocial: string;  
  telephone: string;
  adresse: string;
}

export interface FournissuerUpdateDtoRequest {
  ice?: string;
  raisonSocial?: string;
  telephone?: string;
  adresse?: string;
}